patternlab/styleguide/source/_patterns/03-organisms/by-template/org-nav-contact-row.md

```diff
@@ -0,0 +1,37 @@
+### Description
```

Comment:
This pattern appears to be new, but doesn't have a json file. When I pull it up in Mayflower, I just see a blank screen.